exercises/sieve/test/test_sieve.c

```diff
@@ -0,0 +1,114 @@
+#include <string.h>
+#include "vendor/unity.h"
+#include "../src/sieve.h"
+
+primesArray_t result_array;
+
+// Individual Tests
+void test_no_primes_under_two(void)
+{
+   const unsigned int limit = 1;
+   const primesArray_t expected_prime_array = { 0 };
```

Comment:
Whoops, you're right. I forgot that you need to explicitly initialize one member to get the rest to be implicitly initialized.